config/modules/mock/webpack.js

```diff
@@ -0,0 +1,12 @@
+/*globals module, require */
+const path = require("path");
+const merge = require("webpack-merge");
+const base = require("../base/webpack.base");
+
+module.exports = merge(base({
+    module: "mock"
+}), {
+    output: {
+        path: path.resolve(__dirname, "../../../dist")
```

Comment:
Because mock goes directly to the dist folder, instead the internal module goes in their folder.

(extensions goes in dist/extensions)
(repository goes in dist/repository)
(mock goes in dist)